CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs

```diff
@@ -213,6 +227,8 @@ public void CanSendRequestOnceAtATime()
                
                 _harvestApi.ReceivedRequests.Count.Should().Be(1);
             }
+            
+            
```

Comment:
```suggestion
```

Reply:
resolved in https://github.com/chubberlisk/crypto_tech/pull/30/commits/bf82ffb31fdb96302d5644d8613e869b7814018c